polkadotjs-examples/src/examples/creditcoin-eventsSubscribe.ts

```diff
@@ -0,0 +1,35 @@
+import { ApiPromise, WsProvider, } from '@polkadot/api';
+import type { Vec } from '@polkadot/types-codec';
+
+import {handleTransaction, } from '../utils';
+
+export const registerAddress = async () => {
+    const provider = new WsProvider('ws://localhost:9944');
+    const api = await ApiPromise.create({
+      provider: provider,
+      rpc: {
+        creditcoin: {
+          eventsSubscribe: {
```

Comment:
```bash
$ npx ts-node ./src/examples/creditcoin-eventsSubscribe.ts 
/home/ubuntu/creditcoin-substrate/polkadotjs-examples/node_modules/ts-node/src/index.ts:820
    return new TSError(diagnosticText, diagnosticCodes);
           ^
TSError: ⨯ Unable to compile TypeScript:
src/examples/creditcoin-eventsSubscribe.ts:12:11 - error TS2322: Type '{ params: { name: string; type: string; }[]; type: string; }' is not assignable to type 'DefinitionRpc | DefinitionRpcSub'.
  Type '{ params: { name: string; type: string; }[]; type: string; }' is missing the following properties from type 'DefinitionRpcSub': pubsub, description

12           eventsSubscribe: {
             ~~~~~~~~~~~~~~~
src/examples/creditcoin-eventsSubscribe.ts:32:52 - error TS2339: Property 'creditcoin' does not exist on type 'DecoratedRpc<"promise", RpcInterface>'.

32     const unsubscribe : () => void = await api.rpc.creditcoin.eventsSubscribe(meta, subscriber, (result) => {
                                                      ~~~~~~~~~~
src/examples/creditcoin-eventsSubscribe.ts:32:98 - error TS7006: Parameter 'result' implicitly has an 'any' type.

32     const unsubscribe : () => void = await api.rpc.creditcoin.eventsSubscribe(meta, subscriber, (result) => {
                                                                                                    ~~~~~~
src/examples/creditcoin-eventsSubscribe.ts:35:2 - error TS1005: ',' expected.

35 });
```

@nathanwhit I'm not sure how this method needs to be described here. It's different than `creditcoin.getEvents()`. Can you explain how it works or point me to some example?